math.gobra

```diff
@@ -0,0 +1,62 @@
+// Package math provides common math functions for verification
+package math
+
+// Max returns the maximum of x and y
+ghost
```

Comment:
all functions must be annotated with termination measures, not only the recursive ones. `decreases` without an explicit measure would be valid for these cases